### PR TITLE
OSDOCS#10761: Reorg for "Preparing for install" section

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.17

Issue: https://issues.redhat.com/browse/OSDOCS-10761

Link to docs preview:

Installation methods: https://77702--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/preparing-to-install-on-azure

Preparing to install - IPI: https://77702--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/ipi/installing-azure-preparing-ipi

Preparing to install - UPI: https://77702--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/upi/installing-azure-preparing-upi

QE review:
- [ ] QE has approved this change. NA since this is only a structural change and follows the approved install docs approach

### CODE REVIEW

File: installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service
```
Comment (lahinson) [inline installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc:30]:
```suggestion
* For more information about the Telemetry service, see xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring].
```

File: installing/installing_azure/upi/installing-azure-preparing-upi.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+[id="installing-azure-preparing-upi"]
+= Preparing to install a cluster on Azure
+include::_attributes/common-attributes.adoc[]
+:context: installing-azure-preparing-upi
+
+toc::[]
+
+To prepare for installation of an {product-title} cluster on {azure-short}, complete the following steps:
+
+* You have xref:../../../installing/installing-preparing.adoc#installing-preparing[selected a cluster installation method].
```
Comment (lahinson) [inline installing/installing_azure/upi/installing-azure-preparing-upi.adoc:11]:
```suggestion
* You xref:../../../installing/installing-preparing.adoc#installing-preparing[selected a cluster installation method].
```

File: installing/installing_azure/upi/installing-azure-preparing-upi.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+* You xref:../../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[configured an Azure account] to host the cluster and determined the tested and validated region to deploy the cluster to.
+
+* If you use a firewall, you have xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
```
Comment (lahinson) [inline installing/installing_azure/upi/installing-azure-preparing-upi.adoc:15]:
```suggestion
* If you use a firewall, you xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
```

File: installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+[id="installing-azure-preparing-ipi"]
+= Preparing to install a cluster on Azure
+include::_attributes/common-attributes.adoc[]
+:context: installing-azure-preparing-ipi
+
+toc::[]
+
+To prepare for installation of an {product-title} cluster on {azure-short}, complete the following steps:
+
+* You have xref:../../../installing/installing-preparing.adoc#installing-preparing[selected a cluster installation method].
```
Comment (lahinson) [inline installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc:11]:
```suggestion
* You xref:../../../installing/installing-preparing.adoc#installing-preparing[selected a cluster installation method].
```

File: installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+* You xref:../../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[configured an Azure account] to host the cluster and determined the tested and validated region to deploy the cluster to.
+
+* If you use a firewall, you have xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
```
Comment (lahinson) [inline installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc:15]:
```suggestion
* If you use a firewall, you xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
```

File: installing/installing_azure/upi/installing-azure-preparing-upi.adoc [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service
```
Comment (lahinson) [inline installing/installing_azure/upi/installing-azure-preparing-upi.adoc:30]:
```suggestion
* For more information about the Telemetry service, see xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring].
```

**Review by lahinson — COMMENTED** [review]
Looks good overall! I commented on a couple minor things to fix.

File: installing/installing_azure/upi/installing-azure-preparing-upi.adoc (hunk 2 above)
Comment (snarayan-redhat) [reply installing/installing_azure/upi/installing-azure-preparing-upi.adoc:11]: As per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites), this is a valid usage.

File: installing/installing_azure/upi/installing-azure-preparing-upi.adoc (hunk 3 above)
Comment (snarayan-redhat) [reply installing/installing_azure/upi/installing-azure-preparing-upi.adoc:15]: As per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites), this is a valid usage.

File: installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc (hunk 4 above)
Comment (snarayan-redhat) [reply installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc:11]: As per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites), this is a valid usage.

File: installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc (hunk 5 above)
Comment (snarayan-redhat) [reply installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc:15]: As per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites), this is a valid usage.

File: installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply installing/installing_azure/ipi/installing-azure-preparing-ipi.adoc:30]: Thanks for your review, Laura! :)

File: installing/installing_azure/upi/installing-azure-preparing-upi.adoc (hunk 6 above)
Comment (snarayan-redhat) [reply installing/installing_azure/upi/installing-azure-preparing-upi.adoc]: done

**Pull request merged**